src/cards/cart-item.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Flex, Image, Text, Pressable } from 'native-base';
+import { useNavigation } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { ProductStackParam } from 'src/navigation/product-stack';
+import { DetailedProductData } from 'src/types/products';
+import { AntDesign } from '@expo/vector-icons';
+import { removeFromCart } from 'src/ducks/cart-slice';
+import { useAppDispatch } from 'src/hooks/useful-ducks';
+
+type ProductScreenProps = StackNavigationProp<ProductStackParam, 'Product'>;
+
+interface props {
+    cartData: DetailedProductData;
+}
+
+export const CartItem: React.FC<props> = ({ cartData }) => {
+    const navigation = useNavigation<ProductScreenProps>();
+
+    const dispatch = useAppDispatch();
+
+    return (
+        <Flex
+            width="90%"
+            mt="4"
+            height="280"
+            borderColor="lightBlue.300"
+            backgroundColor="gray.700"
+            direction="row"
```

Comment:
You will also need to ensure that colors are using our theme colors, instead of the default theme colors in nativebase. This way, when you toggle dark and light mode, the component will automatically update its color. Look at the theme file under the constants directory. Most of the time, you will be able to use colors such as `background.[50-900]` or `plainText.[50-900]`